### ISSUE TITLE
Git commit does not reflect in zed interface

### ISSUE THREAD

**Issue by auspy** [zed-industries/zed#14049]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

after commit the files modified stays highlighted. only after clicking on them the highlight of file change is removed.

### Environment

mac m1 air sonoma 14.1.1
Zed 0.142.6

### If applicable, add mockups / screenshots to help explain present your vision of the feature

https://github.com/zed-industries/zed/assets/100942053/5e888d06-2a5e-4ac2-9585-918ad7888794

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by notpeter** [comment]
Perhaps related: 
- https://github.com/zed-industries/zed/issues/13176

**Comment by yongkangchen** [comment]
I tried to reproduce this bug but failed. It would be great if you could provide a screen recording that reflects the whole procedure, including modifying the file, committing it, as well as the status of the tree.

Test on Environment:
mac m1 air sonoma 14.5
Zed 0.142.6

**Comment by jaydenseric** [comment]
This is happening for me too, as far as I can recall for every commit so far.

**Comment by notpeter** [comment]
I'm going to close as a duplicate of:
- https://github.com/zed-industries/zed/issues/13176
